lib/Charge.php

```diff
@@ -79,6 +79,8 @@ public function capture($params = null, $options = null)
      * @param array|null $params
      * @param array|string|null $options
      *
+     * @deprecated Use the `save` method on the Dispute object
+     *
```

Comment:
I'd rather note, as that may have unexpected changes. I think tying the deprecated methods to the deprecated endpoints makes sense.